great_expectations/metrics/domain.py

```diff
@@ -32,6 +39,11 @@ def __new__(cls, *args, **kwargs):
             raise AbstractClassInstantiationError(cls.__name__)
         return super().__new__(cls)
 
+    @classmethod
+    @override
+    def internal_name(cls):
+        return "map"
```

Comment:
I think the name for this is actually `values` isn't it?

Reply:
metric names are the connection point to a metric resolver internally, so the classes we've renamed in this API need to be mapped back to the internal names when we generate the name used for the metric configuration.

Reply:
Where do you see a metric name with the word "map" in it?

Reply:
indeed 🤦‍♂️

Reply:
if `Batch` -> `Table` is the only change we need to make, maybe we should just leave the current magic string transformation in place

Reply:
i reverted this in [0f1e811](https://github.com/great-expectations/great_expectations/pull/10944/commits/0f1e811fa781257221f16c67b4410153328bae54)